### ISSUE TITLE
Windows Peer Review

### ISSUE THREAD

**Issue by AustinSeto** [OpenLiberty/guide-kubernetes-microprofile-config#120]
Peer review for https://github.com/OpenLiberty/guide-kubernetes-microprofile-config/pull/117

Functionality
---
- [x] Check that the guide has working instructions and sample code
- [ ] If any URL visits, try them on different browsers: Firefox, Chrome, Safari
- [x] If any URL visits, try `curl` command where applicable 

Formatting & Presentation
---

`README.adoc` checks:
- [x] Ensure that command substitutions use the `$(cmd)` format and not ``` `cmd` ```
- [x] Ensure that the console output of the integrated tests do not include the `[INFO]` tab
- [ ] Check the quality and presentation of guide according to [Structure and Style Guideline](https://github.com/OpenLiberty/draft-guides-template/wiki/Guidelines-for-Structure-and-Styles)
- [ ] Check that the pom.xml, server.xml, etc files are clean
    - [ ] Remove any dependencies and features that are not required
    - [x] Ensure that the parent pom.xml files do not contain any dependencies, and all dependencies are specified in the children pom.xml files
- [x] Examples with the right outcomes are provided
- [x] Example with the wrong outcomes, if any, are provided

`pom.xml` checks (if files are present):
- [x] Check that the file follows the template found here: https://github.com/OpenLiberty/draft-guides-template/blob/master/finish/pom.xml
- [ ] Check that the following dependencies are included in all of the `pom.xml` files:
```
<!-- Provided dependencies -->
<dependency>
  <groupId>jakarta.platform</groupId>
  <artifactId>jakarta.jakartaee-api</artifactId>
  <version>8.0.0</version>
  <scope>provided</scope>
</dependency>

<dependency>
  <groupId>org.eclipse.microprofile</groupId
  <artifactId>microprofile</artifactId>
  <version>3.3</version>
  <type>pom</type>
  <scope>provided</scope>
</dependency>
```
- [x]  Check that the versions of plugins are directly provided and not fed in by a file property.
  - Not formatted like so:
```
    <properties>
        <!-- Plugins -->
        <version.liberty-maven-plugin>3.2</version.liberty-maven-plugin>
    </properties>

    <!-- Liberty plugin -->
    <plugin>
        <groupId>io.openliberty.tools</groupId>
        <artifactId>liberty-maven-plugin</artifactId>
        <version>${version.liberty-maven-plugin}</version>
    </plugin>
```
  - Should be formatted like so:
```
    <!-- Liberty plugin -->
    <plugin>
        <groupId>io.openliberty.tools</groupId>
        <artifactId>liberty-maven-plugin</artifactId>
        <version>3.2</version>
    </plugin>
```
- [x] Check that there are 4 spaces per indent for proper formatting
- [ ] Ensure that the guide is using the latest version of the `liberty-maven-plugin` or `liberty-gradle-plugin` where applicable

Overall checks:
- [ ] Check the consistency of guide with the template and other guides
- [ ] Check the quality of code according to the best coding practices
- [ ] Check that all licensing statements are properly stated in all files, with the correct year (Should be present in all Java files + the `index.html`)
- [x] Check that the directories are properly structured
- [x] Check that some of these `page-tags` are used in a guide: `MicroProfile, Maven, Docker, Kubernetes, Gradle, Java EE, Security, Cloud`. Only these tags are visible on the website. Latest list [here](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/_includes/visible-tags.liquid). 
- [x] Check the `attribution` statement is accurate for the guide
- [ ] Verify the `travisTest.sh` script, if any, is accurate and consistent with other guides

---

- [ ] Additional tests where applicable: 
    - [ ] Define test coverage and review with team (including guide contributor, if available)
    - [ ] Define detail test cases 
    - [ ] Consider corner cases targeting the specific guide
    - [ ] Consider corner cases UI tests
    - [ ] Consider testing URL on all browsers, ie, FF, Chrome, Safari
    - [ ] Consider testing the `curl` command for URL visits
    - [ ] Make sure the index.html information is correct by visiting the root, e.g. `http://localhost:9080`
- [ ] Consider building with both Maven and Gradle build tools 
- [ ] Testing with different IDEs, ie, Atom, Eclipse (Optional: VS.code, IntelliJ, Microclimate)
- [ ] Run Acrolinx Checker on draft (above 70 score approximately)
- [ ] Consider SEO title and description for the guides 
- [ ] Ensure automated test is enable, set up with Travis CI, and able to schedule tests 
- [ ] Run `diff -r start/ finish/` and there's no differences
- [ ] Ensure that the automation tests are able to run when PR is created
- [ ] Check the appearance of the guide on test site for the following items: 
    - [x] Table of contents 
    - [x] Headings
    - [x] Paragraphs
    - [ ] code snippets
    - [x] outputs
    - [ ] links
    - [x] hotspots 
- [x] Test the guide end-to-end with working instruction and sample code
- [x] Perform all the defined test cases

**Comment by AustinSeto** [comment]
# README

## What you'll learn

- [ ] "any config sources" -> "any configuration sources"

## Deploying the microservices

- [ ] The word "username" is one word

## Updating Kubernetes Resources

- [ ] The description of the `env` section comes before command to replace `kubernetes.yaml`, I think the description should go after the command


# Code

- [ ] Update LMP plugins in `pom.xml` files 

# Other

- [ ] Acrolinx checker run: https://ibmid.acrolinx.cloud/api/v1/checking/scorecards/c067d804-b72a-4836-93f9-f54e73538766
- [ ] Update license years
  - [ ] `README.adoc`

**Comment by griffinhadfield** [comment]
>  There are a number of lines that are exceptionally long. Some have multiple sentences and reach 300 or more characters. These should be broken up into shorter lines.

I will fix that in a seperate PR so that the current changes I have made to the README can still be effectively tracked and seen. Reformatting the lines now may make it appear that I have change nearly the whole README.

**Comment by griffinhadfield** [comment]
Feedback has been addressed

**Comment by AustinSeto** [comment]
One more comment, I noticed in the section `Updating {kube} resources` the hotspots use line numbers. I think this should be replaced with hotspots using the tags.

**Comment by griffinhadfield** [comment]
> One more comment, I noticed in the section `Updating {kube} resources` the hotspots use line numbers. I think this should be replaced with hotspots using the tags.

@AustinSeto Only line number based hotspots seems to work in the `kubernetes.yaml file` not sure why

**Comment by AustinSeto** [comment]
Ok, I think that can be investigated in a later issue. I have seen hotspots used before in `.yaml` files in other guides. All my other changes are addressed after recent commits in #117 so I'll close this now.
